.github/workflows/check-vulnerability-whitesource.yml

```diff
@@ -0,0 +1,45 @@
+name: WhiteSource Vulnerability Scan
+
+on:
+  schedule:
+    - cron: '30 10 * * *'
+  repository_dispatch:
+    types: [check-vulnerability-whitesource]
+
+jobs:
+  whitesource-vulnerability-scan:
+    name: WhiteSource Vulnerability Scan
+    runs-on: ubuntu-18.04
+    timeout-minutes: 80
+    strategy:
+      fail-fast: false
+      matrix:
+      java: [11]
```

Comment:
Line 17 needs to be indented one more level. Make sure your editor is not using tabs.